test/dynamo/test_ctx_manager.py

```diff
@@ -201,19 +210,25 @@ def fn(x):
         self.assertEqual(cnts.frame_count, 2)
         self.assertEqual(cnts.op_count, 9)
 
+    @unittest.expectedFailure  # https://github.com/pytorch/pytorch/issues/118204
```

Comment:
I read the log and found it's a bug that we didn't generate proxy correctly for default ```torch.cuda.Stream```, I can look at it further and fix it.

Reply:
Thanks! I'm going to merge this PR for now to resolve flakiness.